tools/stress-cluster/services/Stress.Watcher/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM mcr.microsoft.com/dotnet/sdk:6.0 AS build
+FROM mcr.microsoft.com/dotnet/sdk:7.0 AS build
```

Comment:
This won't work without updating the stress watcher project to use net7 (csproj, any compile issues, etc.)